cmd/award.go

```diff
@@ -8,27 +8,26 @@ import (
 	"github.com/spf13/cobra"
 )
 
-var recipients []string
-var tags []string
-var message string
-var amount int
-var confirm bool
-var interactive bool
+var (
+	recipients  []string
+	tags        []string
+	message     string
+	amount      int
+	confirm     bool
+	interactive bool
+)
 
 // awardCmd represents the award command
 var awardCmd = &cobra.Command{
 	Use:   "award",
 	Short: "Give away bonuses to your coworkers!",
-	Long: `A longer description that spans multiple lines and likely contains examples
-and usage of using your command. For example:
-
-Cobra is a CLI library for Go that empowers applications.
-This application is a tool to generate the needed files
-to quickly create a Cobra application.`,
+	Long: `This command allows the specification of awards to coworkers.
+You may specify the amount to be given, the destination recipients 
+(who will receive) and the message and hashtags sent along.`,
 	Run: func(cmd *cobra.Command, args []string) {
 		total := len(recipients) * amount
 
-		if validateFlags(amount, tags, recipients, message) == false {
```

Comment:
`validateFlags` returns true if no errors were found, this is why we should check for false before we return. We can't remove the `== false` part or the program will fail.